README.md

```diff
@@ -35,6 +36,9 @@ To build the code without running the tests, add to the "clean build" command th
 > ./gradlew clean build -x test
 ```
 
+### Running pip tests
+Pip tests must run inside a clean pip-environment. Create a virtual environment and provide the path to its '/bin' directory using the 'BITESTS_ARTIFACTORY_PIP_ENV' variable.
+
 ### Running Gradle tests on Artifactory OSS
```

Comment:
The instructions here are incomplete, because they ignore pip and also nuget.

Reply:
Since Artifactory OSS doesn't support pypi and nuget repositories, we won't add it to these tests.